test/iron-ajax.html

```diff
@@ -818,6 +824,39 @@
         });
       });
 
+      suite('when using the bubbles attribute', function () {
+        setup(function() {
+          server.restore();
+        });
+        test('the request and response events should bubble to window', function (done) {
+          server.restore();
+          var total = 0;
+          function incrementTotal(){
+            total++;
+            if(total === 1) done();
```

Comment:
Good catch! I had set it to `1` while testing and forgot to update it to `2`. Just updated it to `2` and tests are still passing. Also fixed the style issues.